application/clicommands/CleanupCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Icinga\Module\Businessprocess\Clicommands;
+
+use Exception;
+use Icinga\Application\Logger;
+use Icinga\Application\Modules\Module;
+use Icinga\Cli\Command;
+use Icinga\Module\Businessprocess\Modification\NodeRemoveAction;
+use Icinga\Module\Businessprocess\ProvidedHook\Icingadb\IcingadbSupport;
+use Icinga\Module\Businessprocess\State\IcingaDbState;
+use Icinga\Module\Businessprocess\State\MonitoringState;
+use Icinga\Module\Businessprocess\Storage\LegacyStorage;
+
+class CleanupCommand extends Command
+{
+    /**
+     * @var LegacyStorage
+     */
+    protected $storage;
+
+    protected $defaultActionName = 'cleanup';
+
+    public function init()
+    {
+        $this->storage = LegacyStorage::getInstance();
+    }
+
+    /**
+     * Cleanup all missing monitoring nodes from the specified config name
+     * If no config name is specified, the missing nodes are cleaned from all available configs.
+     * Invalid config files and file names are ignored
+     *
+     * USAGE
+     *
+     * icingacli businessprocess cleanup [<config-name>]
+     *
+     * OPTIONS
+     *
+     *   <config-name>
+     */
+    public function cleanupAction(): void
+    {
+        $configNames = (array) $this->params->shift() ?: $this->storage->listAllProcessNames();
+
+        $removedNodes = null;
+        foreach ($configNames as $configName) {
+            if (! $this->storage->hasProcess($configName)) {
+                Logger::error('Failed to scan the %s.conf file for missing nodes. File not found', $configName);
+                $removedNodes = false;
```

Comment:
Don't do this. There is no reason not to log that no missing nodes are found. If no config is readable, of course none are found.